mitosheet/mitosheet/step_performers/export_to_file.py

```diff
@@ -97,10 +97,7 @@ def execute(cls, prev_state: State, params: Dict[str, Any]) -> Tuple[State, Opti
             for sheet_index, file_name in sheet_index_to_export_location.items():
                 post_state.dfs[sheet_index].to_csv(file_name, index=False)
```

Comment:
Looks like we write CSV files too - wanna remove that as well? 

Also, let's add a test that no files get creates _until_ the code gets run?

Then - LGTM.